src/GeneralStateTestsFiller/stExample/balanceFiller.json

```diff
@@ -0,0 +1,93 @@
+{
+    "balance" : {
+        "_info" : {
+            "comment" : "A test for balance opcode result"
+        },
+        "env" : {
+            "//comment" : "A blockinfo section (obsolite). Do not change. (only coinbase and gasLimit could be changed)",
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "1000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "//comment" : "Expect section is the post condition for the transaction result",
+        "expect" : [
+            {
+                "//comment" : "indexes are used to specify post condition for a given transaction range.",
+                "//comment" : "transaction section describe multiple transaction which will be executed on pre state",
+                "indexes" : {
+                    "//comment" : "-1 for transaction with any data from this field. or a particular index, or range [0, 1, 2]",
+                    "data" : -1,
+                    "gas" : 0,
+                    "value" : -1
+                },
+                "//comment" : "hard fork order: Frontier, Homestead, EIP150, EIP155, Byzantium, Constantinople, ConstantinopleFix",
+                "network" : [">=ConstantinopleFix"],
+                "result" : {
+                    "//comment" : "Account in expect section would be checked for the fields specified here (balance, code, storage)",
+                    "095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
+                        "//comment" : "Better not to check the gas or mining reward affected account's balances.",
+                        "//comment" : "A change in gasPrice will affect the test and this post condition.",
+                        "//comment" : "So unless the test doesn't really requrie it, don't check coinbase/miner address balance or sender balance in post section",
+                        "balance" : "1000000000000100000",
+                        "//comment" : "Byte Code example",
+                        "code" : "0x73a94f5374fce5edbc8e2a8697c15331677e6ebf0b316000555a600155",
+                        "storage" : {
+                            "0x00" : "0x0de0b6b3a75c5ee0",
+                            "0x01" : "0x0578c0"
```

Comment:
I'm getting `0x057794`, which is exactly `300` less

Reply:
I suspect that this file was generated with `BALANCE` cost at `400`, not increased to `700`, which is why this file has `300` more gas remaining after the execution